sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/Utility.java

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai.implementation;
+
+import com.azure.ai.openai.models.AudioTranscriptionFormat;
+import com.azure.ai.openai.models.AudioTranscriptionOptions;
+import com.azure.ai.openai.models.AudioTranslationOptions;
+import com.azure.core.http.HttpHeaderName;
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.util.logging.ClientLogger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility method class.
+ */
+public final class Utility {
```

Comment:
I have mixed feelings about adding another utitlity class. Also the name is a bit vague. I think that this `Utility` is mixing a couple of things. I would make the following changes:
- Move `getRequestOptionsForMultipartFormData` to the `MultipartDataHelper` class. I think it will be better to not spread this "custom" logic across different places. Since multipart will eventually be provided by `Azure.Core`
- I have mixed feelings about moving the validation step to a separate class. I like that you reduced the code clutter in the `OpenAIClient` and `OpenAIAsyncClient`, but it comes at the price of spreading the business logic into different places. Not to mention, when you get the exception thrown by the `LOGGER` now the reporting class would be `Utility.class` instead of `OpenAIClient.class` or `OpenAIAsyncClient.class` , which adds yet another level of indirection.